### DISCUSSION TITLE
unknown encapsulation rx.cpp

### DISCUSSION THREAD

**Discussion by maxlacourchristensen** [svpcom/wfb-ng#256] [Questions]
Hello,

For ground station on x86 with asus ac56, i try to run

./wfb_rx -K /etc/gs.key -k 8 -n 12 -c 127.0.0.10 -u 5600 -p 0 -l 1000 -e 0 wlx04421a3c8671

First i got: pcap_activate failed: That device is not up

Which I resolved by running:

sudo ip link set wlx04421a3c8671 up

Now I have:

wlx04421a3c8671: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 2312
        inet6 fe80::642:1aff:fe3c:8671  prefixlen 64  scopeid 0x20<link>
        ether 04:42:1a:3c:86:71  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 33 overruns 0  carrier 0  collisions 0

and I get next error:

Error: unknown encapsulation on wlx04421a3c8671

link_encap is 1

I disabled wpa stuff (not showing in ps uaxwww) and put

[keyfile]
unmanaged-devices=interface-name:wlx04421a3c8671

in NetworkManager.conf

such that nmcli shows

wlx04421a3c8671: unmanaged
        "ASUSTek Computer USB-AC56 802.11a/b/g/n/ac"
        wifi (rtl88xxau_wfb), 04:42:1A:3C:86:71, hw, mtu 2312

Can you help?

**Comment by svpcom** [comment]
You need to set link into monitor mode first. See `scripts/rx_standalone.sh` for reference
